packages/mobile/src/harmony-native/components/Avatar/Avatar.tsx

```diff
@@ -38,6 +48,8 @@ export const Avatar = (props: AvatarProps) => {
 
   const { color, shadows } = useTheme()
 
+  const margin = useMargin(other)
```

Comment:
Starting a pattern I borrowed from [styled-system](https://github.com/styled-system/styled-system)

Would like to over time expand these helpers to other props (padding/colors/etc).
Probably would also do a combined hook like 
```tsx
const styles = useThemeStyles(["margin", "color", "padding"])
```